File: java_/Arrayobjects.java
package java_;
class Student {
	public int roll_no;
	public String name;
	Student(int roll_no, String name)
	{
		this.roll_no = roll_no;
		this.name = name;
	}
}

public class Arrayobjects{
	public static void main(String[] args)
	{
		
		Student[] arr;
		arr = new Student[3];
		arr[0] = new Student(1, "vijay");
		arr[1] = new Student(2, "kanth");
		arr[2] = new Student(3, "Chennupati");

		System.out.println(arr.length);
		
		for (int i = 0; i < arr.length; i++){
			System.out.println(arr[i].roll_no + " "+ arr[i].name);
		}
	}
}

File: c++/Arrays/Arrayobjects.cpp
#include<iostream>
using namespace std;

class Student
{

public:
int roll_no;
string name;
Student(int roll_no,string name){
  this->roll_no=roll_no;
  this->name=name;
}
};

int main(){
  Student student[3]={Student(1,"vijay"),Student(2,"kanth"),Student(3,"chennupati")};
  for (int i=0;i<size(student);i++){
  cout << student[i].roll_no<<" "<<student[i].name<<"\n";
  }
  return 0;
}